pokemongo_bot/__init__.py

```diff
@@ -34,10 +34,15 @@ def start(self):
 
     def take_step(self):
         location = self.navigator.take_step()
-        cells = self.find_close_cells(*location[0:2])
+        self.work_cells(1, self.position[0:2])
```

Comment:
Use `i2f(self.api._position_lng)`. To avoid converting all the time, you can create a private method in the Bot main class, like:

```python
@property
def current_coordinates_as_float(self):
    return i2f(self.api._position_lat), i2f(self.api._position_lng) 
```